test/browser/d2l-test.config.js

```diff
@@ -13,8 +10,7 @@ function getGoldenFlag() {
 	};
 }
 
-export default createConfig({
-	pattern,
-	vdiff: true,
+export default {
+	pattern: type => `test/browser/**/*.${type}.js`,
```

Comment:
```suggestion
	pattern: () => `test/browser/**/*.vdiff.js`,
```
Technically this could just be this, since we only use this config file with our vdiff tests here?  Can they name a config file whatever they want and we make that more explicit (`vdiff-config.js`)?